adwaita/variants/base/main_window/headerbar/buttons.css

```diff
@@ -51,13 +51,25 @@
 }
 
 /* Account Button */
+#SteamDesktop .steamdesktop_TopBar_3Z7VQ div[class*="steamdesktop_TitleBarControls_"]
+{
+	flex-direction: row-reverse !important;
+}
+
+#SteamDesktop .steamdesktop_TopBar_3Z7VQ div[class*="steamdesktop_TitleBarControls_"] div[class*="titlebarcontrols_AccountMenu_"]:hover,
+#SteamDesktop .steamdesktop_TopBar_3Z7VQ div[class*="steamdesktop_TitleBarControls_"] div[class*="titlebarcontrols_AccountMenu_"]:active
+{
+	background: none !important;
+}
+
+#SteamDesktop .steamdesktop_TopBar_3Z7VQ div[class*="titlebarcontrols_AccountMenu_"]
 #SteamDesktop .steamdesktop_TopBar_3Z7VQ div[class*="titlebarcontrols_AccountMenu_"],
 #SteamDesktop .steamdesktop_TopBar_3Z7VQ div[class*="titlebarcontrols_AccountMenu_"] div[class*="steamavatar_avatarHolder_"],
 #SteamDesktop .steamdesktop_TopBar_3Z7VQ div[class*="titlebarcontrols_AccountMenu_"] div[class*="steamavatar_avatarHolder_"] img.avatar
 {
 	box-sizing: border-box !important;
-	width: 34px !important;
-	height: 34px !important;
+	width: 28px !important;
+	height: 28px !important;
```

Comment:
Avatar size should probably be increased to 30 or even 32px. And it should be centered in its container because now it looks like there's more padding on the right side.

![image](https://github.com/tkashkin/Adwaita-for-Steam/assets/2003017/f1989888-df46-4823-8d52-96607ba8e263)